rpmconf/Install/zmsetup.pl

```diff
@@ -1475,7 +1475,7 @@ sub setDefaults {
   $config{ldap_dit_base_dn_config} = "cn=zimbra"
     if ($config{ldap_dit_base_dn_config} eq "");
   $config{mailboxd_directory} = "/opt/zimbra/mailboxd";
-  if ( -f "/opt/zimbra/jetty/start.jar" ) {
+  if ( -f "/opt/jetty9/start.jar" ) {
```

Comment:
All looks good to me except question of jetty9. Assuming this can be changed in future?